ios/Video/Features/RCTPlayerObserver.swift

```diff
@@ -151,11 +153,13 @@ class RCTPlayerObserver: NSObject, AVPlayerItemMetadataOutputPushDelegate {
     func addPlayerViewControllerObservers() {
         guard let playerViewController, let _handlers else { return }
 
-        _playerViewControllerReadyForDisplayObserver = playerViewController.observe(
-            \.isReadyForDisplay,
-            options: [.new],
-            changeHandler: _handlers.handleReadyForDisplay
-        )
+        #if !os(visionOS)
+            _playerViewControllerReadyForDisplayObserver = playerViewController.observe(
+                \.isReadyForDisplay,
```

Comment:
According to docs this looks like it's available: https://developer.apple.com/documentation/avfoundation/avplayerlayer/1389748-isreadyfordisplay

Reply:
Thanks for catching this ✅

Reply:
Actually this is not AVPlayerLayer but AVPlayerViewController 😅